crm_lead_to_event_registration/wizard/crm_lead_to_opportunity.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# © 2016 Antiun Ingeniería S.L.
+# © 2016 Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import api, fields, models
+
+
+class CrmLead2OpportunityPartner(models.TransientModel):
+    _inherit = 'crm.lead2opportunity.partner'
+
+    event_id = fields.Many2one(
+        comodel_name="event.event", string="Create registration in this event")
+
+    @api.multi
+    def action_apply(self):
+        res = super(CrmLead2OpportunityPartner, self).action_apply()
+        self.ensure_one()
+        if not self.event_id:
+            return res
+        registration_model = self.env['event.registration']
```

Comment:
Travis:
crm_lead_to_event_registration/wizard/crm_lead_to_opportunity.py:21:9: F841 local variable 'registration_model' is assigned to but never used